tasks/tasks.py

```diff
@@ -156,20 +177,26 @@ async def check_concluded_proposals_task(bot: commands.Bot):
                     logger.error(f"Unknown proposal type: {proposal_type}")
                     continue
 
-                subprocess.run(
-                    [
-                        "node",
-                        "./snapshot/wrapper.js",
-                        title,
-                        proposal_data["draft"]["abstract"],
-                        proposal_data["draft"]["background"],
-                        proposal_data["draft"]["additional"],
-                        "Adopt",
-                        "Reassess",
-                        "Abstain",
-                    ],
-                    check=True,
-                )
+                # Create snapshot proposal by calling the wrapper script
+                proposal_command = [
+                    "node",
```

Comment:
same with this command. would clean things up a bit by moving this to a file specifically for subprocess utilities that need to call JS functions

Reply:
Moved to helpers